Sources/ApodiniREST/RESTExporterConfiguration.swift

```diff
@@ -12,32 +12,35 @@ import Vapor
 extension REST {
     /// Configuration of the RESTful Interface
     public struct Configuration {
-        let configuration: HTTPServer.Configuration
-        let bindAddress: Vapor.BindAddress
+        let bindAddress: Apodini.BindAddress
         let uriPrefix: String
 
-        init(_ configuration: HTTPServer.Configuration) {
-            self.configuration = configuration
-            self.bindAddress = configuration.address
+        init(_ app: Apodini.Application) {
+            // Default initialization
+            if app.http.address == nil {
```

Comment:
As noted in the main review comment I don't think this is the right place to create an default version of the HTTPConfiguration here.